index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
      <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
     </ul>
    </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
```

Comment:
- [ ] For SEO purposes, and considering HTML best practices, your page should have only one `<h1>` tag. With that in mind, you should consider making your `<h1>` tags, and the `<h2>` into `<h3>`. You can read more about headings [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
- [ ] Keep in mind that you should also apply the changes to the following articles.